### PR TITLE
Added send Option to FS20

### PR DESCRIPTION
**Pull request by MK-2001**
Added sendTo Option and send stateChanges on cmdRaw states.
Added some documentation in the Readme.md
Changed Logger to debug to "mute" the adapter with empty logs

### CODE REVIEW

File: cul.js [hunk 1]
```diff
@@ -281,24 +323,27 @@ function connect() {
...
+//}
 
 function main() {
-
-    adapter.getForeignObject('cul.meta.roles', function (err, res) {
+    
+    // in this template all states changes inside the adapters namespace are subscribed
+    adapter.subscribeStates("*");
+    
+    adapter.objects.getObject('cul.meta.roles', function (err, res) {
```
Comment (Apollon77) [inline cul.js]: do not user adapter.objects! This is deprecated! please use the method directly from adapter class and for "getObjectview" also and add a js-controller >=2.1.1 dependency please

File: cul.js [hunk 2]
```diff
@@ -281,24 +323,27 @@ function connect() {
...
         metaRoles = res.native;
-        adapter.getObjectView('system', 'device', {startkey: adapter.namespace + '.', endkey: adapter.namespace + '.\u9999'}, function (err, res) {
+        adapter.objects.getObjectView('system', 'device', {startkey: adapter.namespace + '.', endkey: adapter.namespace + '.\u9999'}, function (err, res) {
```
Comment (Apollon77) [inline cul.js]: see above

File: cul.js [hunk 3]
```diff
@@ -281,24 +323,27 @@ function connect() {
...
             for (var i = 0, l = res.rows.length; i < l; i++) {
                 objects[res.rows[i].id] = res.rows[i].value;
             }
-            adapter.getObjectView('system', 'state', {startkey: adapter.namespace + '.', endkey: adapter.namespace + '.\u9999'}, function (err, res) {
+            adapter.objects.getObjectView('system', 'state', {startkey: adapter.namespace + '.', endkey: adapter.namespace + '.\u9999'}, function (err, res) {
```
Comment (Apollon77) [inline cul.js]: see above

File: io-package.json [hunk 4]
```diff
@@ -1,8 +1,11 @@
 {
     "common": {
         "name": "cul",
-        "version": "1.1.0",
+        "version": "1.1.1",
         "news": {
+            "1.1.1": {
+                "de": "Senden von FS20 cmdRAW möglich oder über sendTo wie in der Readme beschrieben.",
```
Comment (Apollon77) [inline io-package.json]: JSON is invalid. You can use https://translator.iobroker.in/ to auto translate

File: cul.js (hunk 1 above)
Comment (Apollon77) [reply cul.js]: This line should be reverted to getForeignObject

File: cul.js [hunk 5]
```diff
@@ -96,11 +138,11 @@ function checkConnection(host, port, timeout, callback) {
 }
 
 function checkPort(callback) {
-    if (adapter.config.type === 'cuno') {
+    if(adapter.config.type === 'cuno') {
```
Comment (Apollon77) [inline cul.js]: Code was correct here and also regardng the semicolon below according to the "commoon codeing guidelines" ... but I would accept it

File: cul.js [hunk 6]
```diff
@@ -41,6 +58,17 @@ adapter.on('unload', function (callback) {
     callback();
 });
 
+// is called if a subscribed object changes
```
Comment (Apollon77) [inline cul.js]: Why this is added but just logs? If unneeded please remove it

File: cul.js [hunk 7]
```diff
@@ -27,7 +27,24 @@ try {
...
+        adapter.log.debug("my namespace: " + adapter.namespace);
+        adapter.log.debug("State Change " + JSON.stringify(id) + " State: " + JSON.stringify(state));
+        //  State Change "cul.0.FS20.123401.cmd" State: {"val":2,"ack":false,"ts":1581365531968,"q":0,"from":"system.adapter.admin.0","user":"system.user.admin","lc":1581365531968}
+        var oAddr = id.split('.');
+        // 0: cul; 1:0; 2:FS20; 3:123401; 4:cmd;
+        var sHousecode = oAddr[3].substring(0,4);
+        var sAddress = oAddr[3].substring(4,6);
+        switch (oAddr[4]) {
+            case "cmdRaw":
+                sendCommand({protocol: oAddr[2], housecode: sHousecode, address: sAddress, command: state.val});
```
Comment (Apollon77) [inline cul.js]: is only FS20 supported? If yes it should be checked that it is only used for this.

File: cul.js (hunk 1 above)
Comment (GermanBluefox) [reply cul.js]: done

File: cul.js (hunk 2 above)
Comment (GermanBluefox) [reply cul.js]: done

File: cul.js (hunk 3 above)
Comment (GermanBluefox) [reply cul.js]: done

File: cul.js (hunk 5 above)
Comment (GermanBluefox) [reply cul.js]: done

File: cul.js (hunk 7 above)
Comment (MK-2001) [reply cul.js]: Completed with commit https://github.com/ioBroker/ioBroker.cul/pull/11/commits/c980c3373d12a5cc568c23e4479d1b5b0d8da877

**Comment by MK-2001** [comment]
> Thank you, but I have some review comments and it would need some fixes.

All of them are fixed

File: io-package.json (hunk 4 above)
Comment (MK-2001) [reply io-package.json]: done

**Comment by MK-2001** [comment]
Closes #6

**Comment by MK-2001** [comment]
@GermanBluefox Can you tell me when this version will be in the latest available? Can i support it? Does it depends on the checks?

**Pull request merged**